docs/rpc_nodes.md

```diff
@@ -33,7 +29,6 @@ author: Roxane Letourneau
 - ~~Tezos Giga Node from Tezos Ukraine~~ Was expected cease to operations on Oct 1st 2021, but status is unkown. See
   [Announcment](https://twitter.com/GigaNode/status/1435265400699342854)
     - ~~Mainnet: https://mainnet-tezos.giganode.io~~
-    - ~~Granada testnet: https://testnet-tezos.giganode.io/~~
```

Comment:
We can add:
Hangzhou testnet: https://testnet-tezos.giganode.io/

Reply:
added